packages/invoice-dashboard/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@requestnetwork/invoice-dashboard",
-  "version": "0.11.8",
+  "version": "0.11.9",
```

Comment:
Have there been any features merged since the last version? Should this be a minor version bump?

Reply:
Not really, it only has fixes for what is already there.